src/cli/clean.rs

```diff
@@ -162,12 +176,20 @@ async fn clean_cache(args: CacheArgs) -> miette::Result<()> {
     if args.exec {
         dirs.push(cache_dir.join(consts::CACHED_ENVS_DIR));
     }
-    if args.tool {
+    if args.build_backends {
         dirs.push(cache_dir.join(consts::CACHED_BUILD_TOOL_ENVS_DIR));
         // TODO: Let's clean deprecated cache directory.
         // This will be removed in a future release.
         dirs.push(cache_dir.join(consts::_CACHED_BUILD_ENVS_DIR));
     }
+    if args.build {
+        dirs.push(cache_dir.join(consts::CACHED_GIT_DIR));
+        dirs.push(cache_dir.join(consts::CACHED_BUILD_WORK_DIR));
+        dirs.push(cache_dir.join(consts::CACHED_BUILD_BACKENDS));
+        dirs.push(cache_dir.join(consts::CACHED_SOURCE_BUILDS));
+        dirs.push(cache_dir.join(consts::CACHED_SOURCE_METADATA));
+        dirs.push(cache_dir.join(consts::CACHED_PACKAGES));
+    }
```

Comment:
I think this should cover everything?

Reply:
Let's add the `cache_dir.join(consts::CACHED_BUILD_TOOL_ENVS_DIR));` for completeness.



```suggestion
    if args.build {
        dirs.push(cache_dir.join(consts::CACHED_GIT_DIR));
        dirs.push(cache_dir.join(consts::CACHED_BUILD_TOOL_ENVS_DIR));
        dirs.push(cache_dir.join(consts::CACHED_BUILD_WORK_DIR));
        dirs.push(cache_dir.join(consts::CACHED_BUILD_BACKENDS));
        dirs.push(cache_dir.join(consts::CACHED_SOURCE_BUILDS));
        dirs.push(cache_dir.join(consts::CACHED_SOURCE_METADATA));
        dirs.push(cache_dir.join(consts::CACHED_PACKAGES));
    }
```